### ISSUE TITLE
Consider using OAuth over username/passwords for Twitter Actions (and possibly other web based services that supports OAuth)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [biotinker/LibreTasks#89]
```
Just read this on the twitter website when studying the feasibility of OAuth 
for issue 47:

http://apiwiki.twitter.com/Authentication

Basic Authentication sends user credentials in the header of the HTTP request. 
This makes it easy to use, but insecure. OAuth is the Twitter preferred method 
of authentication moving forward - come June 2010, we'll be turning off Basic 
Auth from the API.

There is also a great discussion about using OAuth in mobile apps here:
http://stackoverflow.com/questions/1934187/oauth-secrets-in-mobile-apps

Basically, they are having a similar dilemma I had on issue 47.
```

Original issue reported on code.google.com by `renc...@gmail.com` on 22 Jun 2010 at 3:44

**Comment by biotinker** [comment]
Twitter no longer supported
